templates/quickstart-bitbucket-dc-with-vpc.template.yaml

```diff
@@ -566,6 +566,7 @@ Resources:
         ClusterNodeMax: !Ref 'ClusterNodeMax'
         ClusterNodeMin: !Ref 'ClusterNodeMin'
         CreateBucket: !Ref 'CreateBucket'
+        CustomDnsName: !Ref 'CustomDnsName'
```

Comment:
This was a bug - even when user set `CustomDnsName` parameter in the master template it wasn't propagated.